polylogue/cli/imports.py

```
@@ -48,12 +42,24 @@ def _apply_import_prefs(args: object, env: CommandEnv) -> None:
         return
     if "--html" in import_prefs and getattr(args, "html_mode", "auto") == "auto":
         args.html_mode = "on" if _truthy(import_prefs["--html"]) else "off"
-    if "--attachment-ocr" in import_prefs and _truthy(import_prefs["--attachment-ocr"]):
-        setattr(args, "attachment_ocr", True)
+    if "--attachment-ocr" in import_prefs and not getattr(args, "_attachment_ocr_explicit", False):
+        setattr(args, "attachment_ocr", _truthy(import_prefs["--attachment-ocr"]))
```

Ruff (0.15.9)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)